derive/src/lib.rs

```diff
@@ -44,6 +39,33 @@ struct Args {
 }
 
 impl Args {
+	/// Extracts `eth_abi` argument information from the given `syn::AttributeArgs`.
+	pub fn from_attribute_args(attr_args: syn::AttributeArgs) -> Result<Args> {
+		if attr_args.len() == 0 || attr_args.len() > 2 {
+			return Err(Error::invalid_number_of_arguments(0));
+		}
+		let endpoint_name =
+			if let syn::NestedMeta::Meta(syn::Meta::Word(ident)) = attr_args.get(0).unwrap() {
```

Comment:
I think this auto-formatted piece of code is the best we can have for this particular code or can you make your suggestion more concrete? ^^

Reply:
I mean, in the same way as https://github.com/paritytech/pwasm-abi/pull/74#discussion-diff-226373947R86
```rust 
let endpoint_name = if let syn::NestedMeta::Meta(syn::Meta::Word(ident)) = 
   attr_args.get(0).unwrap() {
```

There is a mix of tabs with whitespaces in formatting also..

Reply:
That's quite a long line then but I can do that if you really want it.

Reply:
I don't think that code has any problems. My opinion is: If it's on the same line it a bit harder to decypher. When it is on a new line it's is immediately visible that it is a if statement and you can see the scope immediately.

Anyway, I believe that both variants are viable, there is a little difference between them, and I would say, it can be treated as a personal style choice. 

I would be really grateful if you won't block merging this PR because of this issue.